### PR TITLE
Get rid of the Visual C++ warnings

### PR DESCRIPTION
**Pull request by afabri**
This PR is work in progress.  It adresses Issue #351.

* [x] Remove unused variables in `catch()`
* [ ] Deal with Eigen warnings

### CODE REVIEW

**Comment by pomerlef** [comment]
ok to test

File: pointmatcher/#IO.cpp# [hunk 1]
```diff
@@ -0,0 +1,2353 @@
+// kate: replace-tabs off; indent-width 4; indent-mode normal
```
Comment (PhiBabin) [inline pointmatcher/#IO.cpp#]: This seem to be a temporary file of your text editor. Please remove it.

File: pointmatcher/.#IO.cpp [hunk 2]
```diff
@@ -0,0 +1 @@
+afabri@MALEVICH.7828:1566821008
```
Comment (PhiBabin) [inline pointmatcher/.#IO.cpp]: same

**Comment by pomerlef** [comment]
@afabri are you still working on this?

**Comment by afabri** [comment]
no.

**Comment by pomerlef** [comment]
Alright, I'll merge that in this state and see later for Eigen's warnings

**Pull request merged**